ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -5439,3 +5439,106 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+# TW-NTC-TPKE
+AHU_DSPM_DFVSC_DTM_RDC_OAMC_OA_CHWRC_RCO2M:
```

Comment:
You are missing a type to cover the economizing capabilities of a typical AHU (e.g., ECON, ECOND)

Reply:
I have added a ECON type.